lib/rules/no-curly-component-invocation.js

```diff
@@ -97,10 +132,11 @@ module.exports = class NoCurlyComponentInvocation extends Rule {
           this.logNode({ message: this.generateError(name), node });
         } else {
           let isExplicitThis = path.this || path.data;
+          let isLocal = this.isLocal(path);
```

Comment:
Should this be `this.isLocal(path.parts[0]);`?

Reply:
the `isLocal()` method figures that out by itself 😉

Reply:
Never mind, I think I read the source of the wrong `isLocal`!